backend/BalanceBuddy/src/main/java/com/example/balancebuddy/config/KeyUtils.java

```diff
@@ -27,79 +27,103 @@ public class KeyUtils {
     @Autowired
     Environment environment;
 
+    // Path to the private key for access tokens
     @Value("${ACCESS_TOKEN_PRIVATE_KEY_PATH}")
     private String accessTokenPrivateKeyPath;
 
+    // Path to the public key for access tokens
     @Value("${ACCESS_TOKEN_PUBLIC_KEY_PATH}")
     private String accessTokenPublicKeyPath;
 
+    // Path to the private key for refresh tokens
     @Value("${REFRESH_TOKEN_PRIVATE_KEY_PATH}")
     private String refreshTokenPrivateKeyPath;
 
+    // Path to the public key for refresh tokens
     @Value("${REFRESH_TOKEN_PUBLIC_KEY_PATH}")
     private String refreshTokenPublicKeyPath;
 
     private KeyPair _accessTokenKeyPair;
     private KeyPair _refreshTokenKeyPair;
 
+    // Retrieves key pair for access tokens, generating it if necessary
     private KeyPair getAccessTokenKeyPair() {
         if (Objects.isNull(_accessTokenKeyPair)) {
             _accessTokenKeyPair = getKeyPair(accessTokenPublicKeyPath, accessTokenPrivateKeyPath);
         }
         return _accessTokenKeyPair;
     }
 
+    // Retrieves the key pair for refresh tokens, generating it if necessary
     private KeyPair getRefreshTokenKeyPair() {
         if (Objects.isNull(_refreshTokenKeyPair)) {
             _refreshTokenKeyPair = getKeyPair(refreshTokenPublicKeyPath, refreshTokenPrivateKeyPath);
         }
         return _refreshTokenKeyPair;
     }
 
+    // Retrieves the key pair from the given public and private key paths.
+    // If the keys do not exist, they will be generated and saved to the specified paths.
     private KeyPair getKeyPair(String publicKeyPath, String privateKeyPath) {
         KeyPair keyPair;
 
+        // Create File objects for the public and private key paths
         File publicKeyFile = new File(publicKeyPath);
         File privateKeyFile = new File(privateKeyPath);
 
+        // Check if both files exist
         if (publicKeyFile.exists() && privateKeyFile.exists()) {
-            //log.info("loading keys from file: {}, {}", publicKeyPath, privateKeyPath);
+            log.info("loading keys from file: {}, {}", publicKeyPath, privateKeyPath);
             try {
+                // Initialize a KeyFactory for the RSA algorithm (RSA user creates and publishes a public key based
+                // on two large prime numbers (that are kept secret), and an auxiliary value. Messages
+                // can be encrypted by anyone, via the public key, but can only be decrypted by someone who knows the private key)

```

Comment:
Just leave "Initialize a KeyFactory for the RSA algorithm". You don't have to explain this in detail especially as they can google the RSA algorithm.